config/application.rb

```diff
@@ -51,13 +51,8 @@ class Application < Rails::Application
 
     config.guidance_url = "https://www.gov.uk/government/publications/additional-payments-for-teaching-eligibility-and-payment-details"
 
-    if ENV["LOGSTASH_HOST"].present?
-      tcp_logger = LogStashLogger.new(type: :tcp,
-                                      host: ENV.fetch("LOGSTASH_HOST"),
-                                      port: ENV.fetch("LOGSTASH_PORT"),
-                                      ssl_enable: true)
-
-      SemanticLogger.add_appender(logger: tcp_logger, level: :info, formatter: :json)
-    end
+    config.log_tags = {
```

Comment:
Where does this magic config come from and where does it get used? Is it picked up by SemanticLogger? Something that LogStash reads? It looks quite mysterious to me know in the context of this PR. Future developer will likely struggle even more to understand it.